ETWAnalyzer/Commands/DumpCommand.cs

```diff
@@ -1221,6 +1229,11 @@ public override void Parse()
                         KeyValuePair<decimal, decimal> minmaxsharedcommit = minmaxsharedcommitMBStr.GetMinMaxDecimal(MiBUnit);
                         MinMaxSharedCommitMiB = new MinMaxRange<decimal>(minmaxsharedcommit.Key / MiBUnit, minmaxsharedcommit.Value / MiBUnit);
                         break;
+                    case "-minmaxworkingsetprivatemib":
+                        string minmaxworkingsetprivateMBStr = GetNextNonArg("-minmaxworkingsetprivatemib");
```

Comment:
minmaxworkingsetprivateMBStr  should be named minmaxworkingsetprivateMiBStr to not confuse the readers about the intended units. MiB are multiples of 1024 * 1024 and MB are multiples of 1000 * 1000.